packages/components/ovh-at-internet/src/utils/index.ts

```diff
@@ -0,0 +1,17 @@
+export const getRandomString = (len: number): string => {
```

Comment:
**question:** Did you consider using `nanoid` instead? (I've seen it from some @frenautvh's PR that's why I'm asking).

Reply:
Could be a great use, this was actually just existing code that a moved. But could be better to avoid rewriting. I'll check that